bundles/org.openhab.ui/web/src/pages/settings/things/channel/channel-copy.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <f7-page @page:afterin="onPageAfterIn" name="channel-copy">
+    <f7-navbar :title="channel.label" :subtitle="thing.label" back-link="Cancel">
+      <f7-nav-right>
+        <f7-link @click="save()" v-if="$theme.md" icon-md="material:save" icon-only />
+        <f7-link @click="save()" v-if="!$theme.md">
+          Done
+        </f7-link>
+      </f7-nav-right>
+    </f7-navbar>
+    <f7-block class="block-narrow">
+      <f7-col v-if="channel">
+        <f7-block-title>Channel</f7-block-title>
+        <channel-general-settings :channel="channel" :channelType="channelType" :createMode="true" />
+      </f7-col>
+      <f7-col v-if="channelType != null">
+        <f7-block-title v-if="configDescription.parameters">
+          Configuration
+        </f7-block-title>
+        <f7-block-footer v-else-if="noConfig" class="padding">
+          This channel has no configuration.<br><br><f7-link back>
+            Go Back
+          </f7-link>
+        </f7-block-footer>
+        <config-sheet
+          :parameter-groups="configDescription.parameterGroups"
+          :parameters="configDescription.parameters"
+          :configuration="config" />
+      </f7-col>
+    </f7-block>
+  </f7-page>
+</template>
+
+<script>
+import ChannelGeneralSettings from '@/pages/settings/things/channel/channel-general-settings.vue'
+import ConfigSheet from '@/components/config/config-sheet.vue'
+
+export default {
+  components: {
+    ChannelGeneralSettings,
+    ConfigSheet
+  },
+  props: ['thing', 'thingType', 'channel', 'channelType', 'channelId', 'createMode'],
```

Comment:
Not all props are used, so they can be removed:

```suggestion
  props: ['thing', 'channel', 'channelType', 'channelId'],
```